conf/claims.routes

```diff
@@ -103,11 +103,11 @@ POST        /multiple/check-movement-reference-numbers              uk.gov.hmrc.
 GET         /multiple/select-duties/:index                          uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.SelectMultipleDutiesController.selectDuties(index: Int)
 POST        /multiple/select-duties/:index                          uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.SelectMultipleDutiesController.selectDutiesSubmit(index: Int)
 
-GET         /multiple/select-duties/:index/:taxCode                 uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.EnterMultipleClaimsController.enterClaim(index: Int, taxCode: TaxCode)
```

Comment:
we are about to split journeys, not to merge them

Reply:
Have reverted changes with my last commit.